core/tests/protractor_utils/DeleteAccountPage.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the delete account page, for use in Protractor
+ * tests.
+ */
+
+var action = require('../protractor_utils/action.js');
+var waitFor = require('./waitFor.js');
+
+var DeleteAccountPage = function() {
+  var DELETE_ACCOUNT_PAGE_URL = '/delete-account';
+  var deleteMyAccountButton = element(
+    by.css('.protractor-test-delete-my-account-button'));
+  var confirmDeletionButton = element(
+    by.css('.protractor-test-confirm-deletion-button'));
+
+  this.get = async function() {
+    await browser.get(DELETE_ACCOUNT_PAGE_URL);
```

Comment:
Add waitFor visibility of any element on this page.

Reply:
Done.

Reply:
Sorry for the confusion, I mean check the visibility of any element on DELETE_ACCOUNT_PAGE_URL page i.e, this function ends once the page is loaded completely!.

Also, remove the first waitFor as we can call this get method from any page and this function should not expect `deleteMyAccountButton` to be present in the parent page, right?

Reply:
Done, this was just my mistake, I put the `waitFor` in the wrong place.